CRM/Financial/Form/Export.php

```diff
@@ -97,7 +97,8 @@ public function preProcess() {
 
     foreach ($batchStatus as $batchStatusId) {
       if ($batchStatusId == $this->_exportStatusId) {
-        CRM_Core_Error::fatal(ts('You cannot exported the batches which were exported earlier.'));
+        $url = CRM_Core_Session::singleton()->readUserContext();
```

Comment:
I don't think you need this $url bit here - check the start of the statusBounce fn

```
  public static function statusBounce($status, $redirect = NULL, $title = NULL) {
    $session = CRM_Core_Session::singleton();
    if (!$redirect) {
      $redirect = $session->readUserContext();
    }
```

Reply:
That's what I thought... but it doesn't work properly without as the user context is not set otherwise in this case.